build.gradle

```
@@ -6,14 +6,16 @@ buildscript {
 
 plugins {
 	id "java"
+	id "java-library"
```

```suggestion
	id "java-library"
```
`java-library` applies `java` (and I think Loom does too)

---

Loom is already using `java`, yes.

---

Java library is where you'll get the api configuration